server/dash/dashboards/neighborhood_recent.py

```diff
@@ -186,7 +171,12 @@ def update_text(selected_council):
 def update_figure(selected_council):
     figure_df = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'").groupby(['createdDate', 'typeName'])['srnumber'].count().reset_index()  # noqa
     figure_df.typeName = figure_df.typeName.map(lambda x: '<br>'.join(textwrap.wrap(x, width=16)))  # noqa
-
+    if figure_df.shape[0] == 0:
+        figure_df = pd.DataFrame(columns=['createdDate', "srnumber", "typeName"])
+        for j in range(len(DISCRETE_COLORS_MAP.keys())):
```

Comment:
since `j` is representing the number of days, maybe we should be using `range(START_DATE_DELTA)`

Reply:
Edited the code to incorporate start date!